### PR TITLE
update builder images to use go1.22.4

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
- update builder images to use go1.22.4

### CODE REVIEW

**Review by Hayden-IO — COMMENTED** [review]
Are we updating https://github.com/sigstore/rekor/blob/main/go.mod#L3 as well?

**Comment by cpanato** [comment]
> Are we updating https://github.com/sigstore/rekor/blob/main/go.mod#L3 as well?

oh let me do that

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/sigstore/rekor
 
-go 1.21
+go 1.22
```
Comment (Hayden-IO) [inline go.mod:3]: can you go mod tidy as well? i assume this will add the toolchain directive

Comment (cpanato) [reply go.mod:3]: i did, and did not updated

**Review by Hayden-IO — APPROVED** [review]

**Pull request merged**